library/core/src/intrinsics.rs

```diff
@@ -2476,6 +2476,7 @@ extern "rust-intrinsic" {
     /// `unreachable_unchecked` is actually being reached. The bug is in *crate A*,
     /// which violates the principle that a `const fn` must behave the same at
     /// compile-time and at run-time. The unsafe code in crate B is fine.
+    #[allow(non_camel_case_types)]
     #[rustc_const_unstable(feature = "const_eval_select", issue = "none")]
     pub fn const_eval_select<ARG: Tuple, F, G, RET>(
```

Comment:
```suggestion
    pub fn const_eval_select<Arg: Tuple, F, G, Ret>(
```
This does look like a clear style violation to me.